front/src/views/profile/accommodation/accommodation.view.html

```diff
@@ -43,4 +79,6 @@ <h6 class="accent--text text--lighten-2 text-uppercase">
       <div class="map" ref="map-holder"></div>
     </div>
   </div>
+  <v-pagination :length="paginationInfo.pageCount || 0" v-model="changedPage" color="accent lighten-1"
+    @input="updateQuery" />
 </div>
```

Comment:
new line

Reply:
Ahah. yes.
But eslint formatter doesn't think so :*(

Reply:
this can't be happening. maybe he's swearing at extra spaces instead of a new line at the end?)